parsec/core/fs/remote_loader.py

```diff
@@ -87,12 +87,14 @@ def __init__(
         device: LocalDevice,
         workspace_id: EntryID,
         get_workspace_entry: Callable[[], WorkspaceEntry],
+        get_previous_workspace_entry: Callable[[], Awaitable[WorkspaceEntry]],
```

Comment:
We should document the exception this function could raise
Ideally we should have `get_workspace_entry` and `get_previous_workspace_entry` never raise any exception (only returning `None` if the workspace entry cannot be found), so a single comment here would do the trick

However I guess it's hard to have `get_previous_workspace_entry` not raising any exception (because if it call ends up with a `FSBackendOfflineError` and we return a `None`, then we are most likely going to give the wrong information to the user...)

So maybe we should allow `FSError`, but this should be documented, and given documentation of a callback is messy, we should also document the call site that only an `FSError` might be raised.

Reply:
5769bda